common/lib/xmodule/xmodule/course_module.py

```diff
@@ -399,7 +407,7 @@ class CourseFields(object):
         scope=Scope.settings,
         default=[
             {
-                "short_description": _("Getting Started With Studio"),
+                "short_description": _("Getting Started With {studio_name}").format(studio_name=settings.STUDIO_NAME),
```

Comment:
Just double-checking that we set settings.STUDIO_NAME in both apps (lms & cms), yes?